### PR TITLE
BETA: Register jurgen.is-a.dev

### PR DESCRIPTION
**Pull request by def0e0**
Added `jurgen.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/jurgen.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "def0e0",
+        "email": "def0e0@yandex.ru"
+    },
+    "record": {
+        "A": ["45.148.120.75"]
```
Comment (Stef-00012) [inline domains/jurgen.json:7]: I cannot reach your website
> 404 Not Found
> nginx/1.18.0 (Ubuntu)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/jurgen.json (hunk 1 above)
Comment (def0e0) [reply domains/jurgen.json:7]: @Stef-00012 Oops. I fixed it.

Comment (Stef-00012) [reply domains/jurgen.json:7]: now i can access it

**Comment by Stef-00012** [comment]
seems good now

**Review by iostpa — APPROVED** [review]

**Pull request merged**